### PR TITLE
Added login, signup and forgotten password for auth

### PR DESCRIPTION
**Pull request by Akhigbepeace**
- Added login and forgotten password pages for admin authentication
- Added other pages that needs to be editable

### CODE REVIEW

File: src/pages/login.tsx [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+        position="relative"
+      ></Box>
+
+      <Box
+        flexDirection="column"
+        alignItems="center"
+        mt="-80px"
+        borderRadius="5px"
+        p="30px"
+        w={{
```
Comment (chidioguejiofor) [inline src/pages/login.tsx:69]: Can you create a dumb component called "SignUpLayout" abstract the UI jsx that is contained in this file

File: src/pages/forgotPassword.tsx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        borderRadius="5px"
+        p="30px"
+        w={{
+          sm: "290px",
+          md: "400px",
+          lg: "500px",
+          xl: "500px",
+          "2xl": "500px",
+        }}
+        mx="auto"
```
Comment (chidioguejiofor) [inline src/pages/forgotPassword.tsx:50]: Similar to the previous comment, create a "ForgotPasswordLayout" component.

File: src/pages/admin/downloads.tsx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import Downloads from "pageLayouts/Downloads";
+
+const DownloadPage = () => {
```
Comment (chidioguejiofor) [inline src/pages/admin/downloads.tsx:3]: I am guessing you added these `tsx` files but did not delete the `jsx` right? If that is the case, feel free to  delete the `jsx` in this PR

**Review by chidioguejiofor — COMMENTED** [review]
Dropped some comments. Please take a look at them. I want us to complete all these quickly

File: src/pages/login.tsx (hunk 1 above)
Comment (Akhigbepeace) [reply src/pages/login.tsx:69]: I'd be doing the same for the login right ?

**Review by chidioguejiofor — APPROVED** [review]

**Pull request merged**